.github/workflows/ci.yml

```diff
@@ -46,32 +46,6 @@ jobs:
           ./script/lints/lint_fixtures.sh
           ./script/lints/lint_scripts.sh
 
-  prism:
```

Comment:
?

Reply:
At first I was just gonna have a prism-specific job to run tests, but instead I folded it all into actual explicit tests, so we don't need the separate job anymore

Reply:
That is to say: the prism fixtures are run in `tests/prism_fixtures_test.rs` now using the `--prism` flag instead of some spooky environment variable shenanigans

Reply:
cool sg